### ISSUE TITLE
Can't seem to get it to do anything.

### ISSUE THREAD

**Issue by VagishVela** [avaidyam/Parrot#23]
This is all I get, I logged in. Maybe it is because I have a project fi enabled hangouts account?
<img width="689" alt="screen shot 2016-04-02 at 10 48 33 am" src="https://cloud.githubusercontent.com/assets/7334882/14227011/b040e88a-f8c0-11e5-8cf9-14997819c07f.png">

**Comment by avaidyam** [comment]
Can you check your `Console.app` log and see if anything's there? I'm pretty sure Project Fi shouldn't mess with Parrot. I could be wrong.

**Comment by VagishVela** [comment]
It opens but doesn't even load that page anymore. I don't know what messages to look for in Console.app

**Comment by avaidyam** [comment]
That's weird. Look for `Parrot.app` or anything `Parrot` related?

**Comment by avaidyam** [comment]
I have a Google Voice, not a Project Fi, but it should be the same...

**Comment by avaidyam** [comment]
@VagishVela I'd like to ask: are you building the source yourself or using the distribution binary?

**Comment by VagishVela** [comment]
@avaidyam The distribution binary.

**Comment by avaidyam** [comment]
Let me build a new one -- that one could possibly be out of date.

**Comment by avaidyam** [comment]
@VagishVela Any progress?

**Comment by VagishVela** [comment]
@avaidyam I haven't tested it since. Let me try again later today.

**Comment by avaidyam** [comment]
@VagishVela There's a new release available that you can try. I'm closing this issue for now.
